packages/lesswrong/lib/collections/users/views.ts

```diff
@@ -270,3 +270,16 @@ Users.addView("walledGardenInvitees", function () {
   }
 })
 ensureIndex(Users, {walledGardenInvite: 1})
+
+Users.addView("usersWithOptedInToDialogueFacilitation", function (terms: UsersViewTerms) {
+  return {
+    selector: {
+      optedInToDialogueFacilitation: true
+    },
+    options: {
+      sort: {
+        karma: -1
```

Comment:
Not sure how performant this will be if we don't have an index on either `optedInToDialogueFacilitation` or `karma`

Reply:
Leaving for discussing with Jacob

Reply:
added